chartfx-dataset/src/test/java/de/gsi/dataset/spi/MultiDimDoubleDataSetTests.java

```diff
@@ -0,0 +1,105 @@
+package de.gsi.dataset.spi;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Tests for the MultiDimDoubleDataSet
+ * TODO: Test EventListeners
+ * 
+ * @author Alexander Krimm
+ */
+class MultiDimDoubleDataSetTests {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MultiDimDoubleDataSetTests.class);
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5084368)